Templates/Class Template/q2.cpp

```diff
@@ -0,0 +1,31 @@
+#include <iostream>
```

Comment:
Can you add the question as comment on the top?

Reply:
Ok I will add the question and update the README.md file as well.